bitnami/redis/templates/configmap.yaml

```diff
@@ -1,53 +1,59 @@
 apiVersion: v1
```

Comment:
I would allow an existingConfigmap

Reply:
I saw that there is a value called existingConfigmap, please add it

Reply:
That was the original plan